akka-persistence-typed/src/test/java/jdocs/akka/persistence/typed/PersistentFsmToTypedMigrationCompileOnlyTest.java

```diff
@@ -0,0 +1,211 @@
+/*
+ * Copyright (C) 2019 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package jdocs.akka.persistence.typed;
+
+import akka.actor.typed.ActorRef;
+import akka.persistence.typed.*;
+import akka.persistence.typed.javadsl.*;
+
+import static akka.persistence.fsm.AbstractPersistentFSMTest.WebStoreCustomerFSM.*;
+
+public class PersistentFsmToTypedMigrationCompileOnlyTest {
+
+  //#commands
+  interface Command {}
+
+  public static class AddItem implements Command {
+    public final Item item;
+
+    public AddItem(Item item) {
+      this.item = item;
+    }
+  }
+
+  public static class GetCurrentCart implements Command, ExpectingReply<ShoppingCart> {
```

Comment:
If one is not using EnforcedReplies the `ExpectingReply` trait is not adding much value. For this example I think we should not use it, to reduce the number of new concepts to learn. If they already used PersistentFSM they are used to replying to sender and then it should not be a problem to send the reply from `andThen`.

Reply:
Yep will remove